backend/backend.yaml

```diff
@@ -0,0 +1,601 @@
+# send everything to http://otel-collector.observability-backend.svc.cluster.local:4317
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: observability-backend
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: collector-config
+  namespace: observability-backend
+data:
+  collector.yaml: |
+    receivers:
+      otlp:
+        protocols:
+          grpc:
+      prometheus:
```

Comment:
Why there is p8s receiver?

Reply:
It is there to get metrics from the input of the data to the backend service. Which is this single open telemetry collector running as "gateway".

For example, in my test with petclinc, this proved to be very helpful in checking whether log data are transmitted at all. It turned out that this was not the case.

Reply:
> It is there to get metrics from the input of the data to the backend service. 

sorry but I don't understand. Anyways it seems like it is just scraping itself/the collector..

Reply:
Right, but all signals pass that collector when you send them to the obs backend service address.